.github/workflows/release.yml

```diff
@@ -0,0 +1,85 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+      - next
+      - next-major
+      - beta
+      - alpha
```

Comment:
This is only configuration. It will be good to have this already in place, and ONLY create this branch on the project when it is needed or there are reason for.